scripts/deploy

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+
+build_chart() {
+  if [[ -z "${DOCKERHUB_USER}" ]]; then
+    echo "Please set env DOCKERHUB_USER to your dockerhub username"
+    exit 0
+  fi
+  #In the case of trying to build on non amd64 hosts, docker-buildx allows you to
+  #Cross compile for different architectures.
+  if [ "$ARCH" != "amd64" ]; then
```

Comment:
I would specifically make this for M1 arm, not `not amd64`

Reply:
I thought we agreed to have it standardly build for amd64. So if the arch is not amd64 it needs to be able to build for the set platform. Wouldn't it always need to be compiled for amd64 when its not amd64?